docs/components/custom-dns-server.rst

```diff
@@ -1,6 +1,8 @@
 Custom DNS Server
 -----------------
 
+.. versionadded:: 2.3.0
```

Comment:
Ouch, I can't even remember this. How much time did it took? 😱

Reply:
:rofl: ohh how old is this typo :eyes: